chart/values.yaml

```diff
@@ -10,16 +10,47 @@ istio:
 federation:
   meshPeers:
     local:
+      # Name is a unique identifier of the peer used as its service name suffix.
+      # Defaults to the helm release name.
+      # name: "east"
       controlPlane:
         # Local control plane namespace is used to create local Istio configs (ServiceEntry for imported services,
-        # Gateway for exported services, VirtualService and DestinationRule for routing to federation-discovery-service).
+        # Gateway for exported services, etc.).
         namespace: istio-system
+      gateways:
+        ingress:
+          # Ingress gateway selector specifies to which workloads Gateway configurations will be applied.
+          # selector:
+          #   app: federation-ingress-gateway
+          port:
+            # Port name of the ingress gateway Service.
+            # This is relevant only when the ingressType is openshift-router, but it cannot be empty.
```

Comment:
`port.name` is used in the `Gateway` resource managed by the controller, and if `openshift-router` is enabled, this port name is also used in `Route.spec.port.targetPort` - then the route target port must match the gateway's service port name. If the ingress type is Istio, only the port number is important and the name could be whatever you want, because it's not used by Istio.

Reply:
Makes sense. Thanks for the clarification.